server/src/db/migrations/20210101210207_createUsers.cjs

```diff
@@ -17,6 +17,8 @@ exports.up = async (knex) => {
       table.bigIncrements("id");
       table.string("email").notNullable().unique();
       table.string("cryptedPassword").notNullable();
+      table.string("firstName").notNullable()
+      table.string('lastName').notNullable()
```

Comment:
we should not edit existing migrations, and instead create a NEW migration file with the changes we want (like adding a new column). Remember that when the app was created and the repo was added to github that this original User migration has already been run. That means that the knex database has already looked at the timestamp of this migration file name. When we run `migrate:latest`, knex will ignore this file and thus not add the new columns you have specified.

If we leave this modification, each group member will have to drop their database on their machine, re-create the database, and run the migrations again.
Alternatively, the better solution is to always make a new migration file for any changes to tables in the database (adding tables, adding column to a table, removing a column from a table, renaming a column, etc). The changes are tacked on that new migration file, so when the PR is merged in each group member just needs to run the new migration with `migrate:latest`